docker/Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM debian:buster-20221219-slim
 
 ARG TARGETARCH
 ARG HATCH_VERSION=1.8.1
+ENV SCCACHE_VERSION=0.8.1
```

Comment:
do we want just set `DD_USE_SCCACHE=1` here?

Do we also want to set a different/explicit sccache directory? since I assume we'll use a volume for the cache in the docker compose?

Reply:
oh, thought it's turned on automatically if `sccache` binary is there. Good point. 

Hmm, yeah we'll be using the volume for the cache in the docker compose, instead of host one and possibly persisting over separate invocations of scripts/ddtest